nautobot/extras/models/jobs.py

```diff
@@ -594,6 +593,8 @@ class JobResult(BaseModel, CustomFieldModel):
 
     objects = JobResultManager()
 
+    _natural_key = ["task_id"]
```

Comment:
This is required because we told it to not include `UUIDField` by default, right? Can you please add a brief comment for that?

Reply:
Addressed by instead reworking the logic so that it doesn't automatically exclude UUIDField, only `id`.